extensions/amp-dailymotion/1.0/test/validator-amp-dailymotion.html

```diff
@@ -0,0 +1,40 @@
+<!--
```

Comment:
There should be a matching .out file generated for this test.

Reply:
@rebeccanthomas Since this is the Preact implementation, let's remove AMP-mode related files for now. This includes validation files, and `amp-dailymotion.js`/`base-element.js`.